client/src/containers/AuthContainer.css

```diff
@@ -1,5 +1,8 @@
 .AuthContainer {
-  background-color: #488fe0;
+  background: url("https://cdn.skyrisecities.com/sites/default/files/images/projects/6977/6977-78314.jpeg");
```

Comment:
Do we have a licence?

Reply:
Couldn't find any good pictures on unsplash, is it fine if we leave this as a background for now since we're still in development and aren't going to be releasing it anytime soon?

Reply:
The image was taken by University of Toronto, our project is for University of Toronto.. I think we may be allowed to use it

Reply:
I don't think we are. Despite the project being a UofT project, we would still need permission to use their assets. This could raise some questions during demo at the very least.